.github/workflows/anghabench-after-build.yml

```diff
@@ -4,7 +4,7 @@ on:
   # Only run when normal CI steps complete
   # Otherwise we'd just fail to build this anvill branch
   workflow_run:
-    workflows: ["Build"]
+    workflows: ["Build On PR"]
```

Comment:
I think this should be reverted since the main workflow name didn't change. Am I missing something?

Reply:
Yes this looks like an accident